src/ticks.jl

```diff
@@ -201,8 +207,8 @@ function optimize_ticks_typed(x_min::T, x_max::T, extend_ticks,
                         end
                         # round only those values that end up as viewmin and viewmax
                         # to save computation time
-                        S[k + 1] = round(S[k + 1], sigdigits = sigdigits(z))
-                        S[2 * k] = round(S[2 * k], sigdigits = sigdigits(z))
+                        S[k + 1] = round(S[k + 1], sigdigits=sigdigits(z), base=Int(base))
```

Comment:
Does this not fail for `base=ℯ`?

Reply:
It does, good catch ! Rounding using the `base` keyword requires an Integer. So what I propose is to use the base kw for `log2` and `log10 `and omit the kw for `log` ?